bin/config.ru

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+require 'rack'
+require 'pry-byebug'
+$LOAD_PATH << File.expand_path("../../lib", __FILE__)
```

Comment:
Enrique was telling me that 

`lib = File.expand_path("../../lib", __FILE__)
$:.unshift(lib)` 
is a nicer way to do the load path. $: is shorthand for $LOAD_PATH